.travis.yml

```diff
@@ -132,7 +132,8 @@ jobs:
   # Unit tests (including test runs for benchmarks).
   - name: unit-test
     script:
-    - cargo test --all --benches
+    - cargo test --all
+    - cargo test --benches
```

Comment:
This will run lib tests again in addition to bench tests. AFAIK, the way to deal with this behavior of `cargo test` is to run `cargo test --all --lib --tests --benches` (i.e., everything except for doc tests) and separately `cargo test --all --doc` (`--doc` cannot be combined with other test categories).

Reply:
Hmm... Maybe then just `cargo bench --lib` instead of `cargo test --benches`? Or it will do something different?

Reply:
I think `cargo bench --lib` runs benchmarks rather then just tests it.